#!/usr/bin/env python
PACKAGE = "robocup_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('mapping_margin', double_t, 0, "mapping margin", 0.10, 0, 5)

exit(gen.generate(PACKAGE, "robocup_map_server", "robocup_map_server_dynamic_params"))
